index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fitness</title>
+    <link rel="stylesheet" href="main.css" />
+    <script
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="desk_nav">
+        <div class="list-1">
+          <ul>
+            <i class="fa-brands fa-facebook"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <li><a href="#">Logout</a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">Somali</a></li>
+          </ul>
+        </div>
+        <nav class="nav">
+          <div lgo>
+            <a href="index.html">
+              <img src="./img/logo.png" alt="" class="logo"
+            /></a>
+          </div>
+          <ul>
+            <a href="./about.html">About</a>
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">Updates</a>
+            <h4 class="list-h4">BB compaign</h4>
+          </ul>
+        </nav>
+      </div>
+      <div class="mobile-menu">
+        <li class="menuBtn">
+          <i class="fa-solid fa-bars fa-2x"></i>
+        </li>
+
+        <div class="menu hidden">
+          <ul>
+            <a href="about.html" onclick="closeMenu()">About</a>
+            <a href="#" onclick="closeMenu()">Program</a>
+            <a href="#" onclick="closeMenu()">Join us</a>
+            <a href="#" onclick="closeMenu()">Sponsor</a>
+            <a href="#" onclick="closeMenu()">Updates</a>
+          </ul>
+        </div>
+        <li class="close-menu hidden"><i class="fa-solid fa-x fa-2x"></i></li>
+      </div>
+    </header>
+    <section>
+      <div class="description">
+        <h3 class="desc-header">"Hello! body lovers"</h3>
+        <h1>
```

Comment:
- [ ] @husush23 great work implementing the changes requested, however, your design style has some sections not following the design guidelines given:  the text color and background don't match the one given
![image](https://user-images.githubusercontent.com/61408860/234608705-047bb1f7-d7b4-4c13-baf8-d4f619edbec4.png)



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/234607747-f992a53f-5176-4b0a-8574-78f1b069cc32.png)  |  ![image](https://user-images.githubusercontent.com/61408860/234608000-648efe38-017e-4f58-9de2-4133ec41f786.png)